packages/nextjs/test/run-integration-tests.sh

```diff
@@ -44,7 +44,7 @@ for NEXTJS_VERSION in 10 11 12; do
   fi
 
   # Next.js v11 requires at least Node v12
-  if [ "$NODE_MAJOR" -lt "12" ] && [ "$NEXTJS_VERSION" -eq "11" ]; then
+  if [ "$NODE_MAJOR" -lt "12" ] && [ "$NEXTJS_VERSION" -ge "11" ]; then
     echo "[nextjs$NEXTJS_VERSION] Not compatible with Node $NODE_MAJOR"
     exit 0
   fi
```

Comment:
Can we add another check like this making sure that if we're testing Next 13, Node is at least version 14?